app/controllers/bookings_controller.rb

```diff
@@ -22,6 +22,16 @@ def create
   end
 
   def index
+    @category = {
+      "strings" => "https://images.unsplash.com/photo-1460036521480-ff49c08c2781?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=2704&q=80",
+      "guitar" => "https://images.unsplash.com/photo-1525201548942-d8732f6617a0?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2700&q=80",
+      "keyboard" => "https://images.unsplash.com/photo-1562166622-962955595c6a?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2700&q=80",
+      "percussion" => "https://images.unsplash.com/photo-1543443283-8f4bb9683f54?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2700&q=80",
+      "brass" => "https://images.unsplash.com/photo-1511192336575-5a79af67a629?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=3578&q=80",
+      "woodwind" => "https://images.unsplash.com/photo-1606139510019-a3e0bbc94c57?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2698&q=80",
+      "equipment" => "https://images.unsplash.com/photo-1516575869513-3f418f8902ca?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1233&q=80",
+      "turntable" => "https://images.unsplash.com/photo-1485726696757-76885c99f0f5?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2000&q=80"
+    }
```

Comment:
yeah i thought of that too but when we upload later we will only use users' uploaded photo instead of default pictures no? in that case i will just delete these when we do

Reply:
if user doesn't upload picture, those will be default